public/injectEnv.js

```diff
@@ -2,7 +2,7 @@ window.injectedEnv ={
 	REACT_APP_BACKEND_GETUSERINFO_API:'https://k9dc.essential-dev.com/fence/login/',
```

Comment:
what are the changes that made in package.json? 30000 line?

Reply:
No change in package.json

Reply:
Sorry I mean package-lock?